src-plugins/itkFilters/itkMorphologicalFiltersToolBox.cpp

```diff
@@ -65,14 +66,14 @@ itkMorphologicalFiltersToolBox::itkMorphologicalFiltersToolBox ( QWidget *parent
     d->filters->addItems ( filtersList );
 
     // We use the same widget for all the morphological filters
```

Comment:
From here to line 92, I only changed the variable names to ease readability since I added other widgets in the toolbox.